src/main/java/com/archisacadeny/course/Course.java

```diff
@@ -1,7 +1,38 @@
 package com.archisacadeny.course;
 
 import com.archisacadeny.config.DataBaseConfig;
+import com.archisacadeny.instructor.Instructor;
+import com.archisacadeny.student.Student;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class Course {
+    private long id;
+    private String courseName;
+    private Instructor instructor;
+
```

Comment:
we should add other fields for this class such as:
credit
total student
enrolledStudents
department

Reply:
Already added